src/WebJobs.Script.WebHost/Web.config

```diff
@@ -43,7 +43,7 @@
       <remove name="ExtensionlessUrlHandler-Integrated-4.0" />
       <remove name="OPTIONSVerbHandler" />
       <remove name="TRACEVerbHandler" />
-      <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />

```

Comment:
>="*." [](start = 61, length = 5)

This needs more discussion. 

Look at the issue [https://github.com/Azure/azure-webjobs-sdk-script/issues/969](https://github.com/Azure/azure-webjobs-sdk-script/issues/969)

Essentially, Functions currently does not allow paths to have a period in them. for instance /index.html, etc etc. Proxies does allow users to have such paths. With Proxies merging with function this creates a problem. To enable such paths, we need to ensure that the manage module are executed for such path. There are a couple of way of enabling this as mentioned in the issue.

The change made here is a way of ensuring that TransferRequestHandler is invoked for all requests.